### PR TITLE
Add documentation for Idle Detection API interfaces

### PR DESCRIPTION
**Pull request by reillyeon**

### CODE REVIEW

File: content/en-US/api/IdleDetector.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+**`IdleDetector.screenState`**
+
+Returns either `"locked"` if the device's screen is locked or `"unlocked"` if it
+is not. This attribute will return `null` before `start()` is called.
+
+## Events
+
+**`IdleDetector.onchange`**
+
+Called when the value of `userState` or `screenState` has changed.
```
Comment (jpmedley) [inline content/en-US/api/IdleDetector.md]: What event type is passed to this callback?

**Review by jpmedley — CHANGES_REQUESTED** [review]
There's only one issue. It's about the event callback. 

Did the origin trial relaunch in 88?

File: content/en-US/api/IdleDetector.md (hunk 1 above)
Comment (reillyeon) [reply content/en-US/api/IdleDetector.md]: This event uses the generic `Event` type. How should I phrase that here?

**Comment by reillyeon** [comment]
> Did the origin trial relaunch in 88?

I have requested that the Origin Trial resume in M-88. I haven't gotten confirmation yet that it has been enabled.

File: content/en-US/api/IdleDetector.md (hunk 1 above)
Comment (jpmedley) [inline content/en-US/api/IdleDetector.md]:
```suggestion
Called when the value of `userState` or `screenState` has changed. This method receives an `Event` object.
```

Comment (jpmedley) [reply content/en-US/api/IdleDetector.md]: My answer is in the adjacent suggestion.

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by jpmedley** [comment]
@reillyeon Do you have any time to finish this and #19

**Comment by reillyeon** [comment]
Please take another look.

**Pull request merged**